README.md

```diff
@@ -291,6 +291,7 @@ linux | Linux Guest
 linux.efi | Linux EFI Guest
 opensuse.leap | OpenSUSE Leap
 opensuse.tumbleweed | OpenSUSE Tumbleweed
+oracle.linux | Oracle Linux
```

Comment:
Again, something for a follow up but for the multi-arch preferences it would be good to list the supported arches here.

Reply:
What about naming this `oraclelinux` instead?

Reply:
btw. isn't this generated from `make readme`?